### ISSUE TITLE
[BUG] Unclear that error has occurred in UI

### ISSUE THREAD

**Issue by BenElgar** [deref/exo#107]
**Describe the bug**
When an `exo run` command fails it is unclear that this has happened in the UI.

**To Reproduce**
1. Create a docker-compose file with an error, e.g.
```yaml
services:
  alpine:
    image: "alpine"
    command: "notacommand"
```

That will give an error like the following in the server but no error in the UI.
```
19:47:31 server 2021/08/06 18:47:31 starting container "73d8bd1d97b21c1c00d94a9b9b83fbc985829e3045597e8af9c223fd15f2cbf3": Error response from daemon: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "notacommand": executable file not found in $PATH: unknown
```

**Expected behavior**
Some error message in the UI

**Comment by brandonbloom** [comment]
Working on this as part of #105

**Comment by kendru** [comment]
Closing as duplicate of/fixed by #105
